test/Testing.js

```diff
@@ -0,0 +1,153 @@
+
+// var db = require('./db');
+// var heroesClass = require('./heroes');
+// var assert = require('assert');
+
+const chai = require('chai'),
+	chaiHttp = require('chai-http'),
+	mocha = require('mocha'),
+	sinon = require('sinon'),
+	db = require('../db'),
+	heroesClass = require('../heroes'),
+	server = require('../server'),
+	assert = chai.assert;
+
+chai.should();
+chai.use(chaiHttp);
+
+describe('Heroes', function()
+{
+	describe('#getHeroes()', function()
+	{
+		it('should return all heroes', function()
+		{
+			const heroes = new heroesClass.Heroes();
```

Comment:
in tests we never use real data because real data can change.
instead we use mock.